src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/ciagkpi/PefCiagKpiService.kt

```diff
@@ -31,39 +37,76 @@ class PefCiagKpiService(
   private val telemetryService: TelemetryService,
   private val timelineService: TimelineService,
   private val timelineEventFactory: TimelineEventFactory,
+  private val reviewService: ReviewService,
 
 ) : CiagKpiService() {
 
+  /**
+   * Process an prisoner admission.
+   *
+   * - If the prisoner is brand new create a new Induction schedule
+   * - If the prisoner already has an incomplete induction schedule (this will most likely be
+   * a duplicate message) then do nothing.
+   * - If the prisoner already has had their Induction created. Then this person is likely to
+   * be a coming back into prison and needs to resume their Reviews.
+   */
   override fun processPrisonerAdmission(prisonNumber: String, prisonAdmittedTo: String, eventDate: Instant) {
```

Comment:
The changes in here look scarier than they are. Basically split this function in to more discreet functions. Also an explanation of whats happening in the java doc (kotlin doc sounds weird)